lib/cuke_linter/helpers/dialect_helper.rb

```diff
@@ -0,0 +1,25 @@
+module CukeLinter
+  module DialectHelper
+
+    def self.set_dialect(dialect)
+      @dialect = dialect
+    end
+
+    def self.given_keyword
+      @dialect['given'][1].strip
+    end
+
+    def self.when_keyword
+      @dialect['when'][1].strip
+    end
+
+    def self.then_keyword
+      @dialect['then'][1].strip
```

Comment:
It looks like these methods only get one keyword per type. Any particular keyword can have several synonyms and linters will need to be able to work with any of them. Perhaps return the whole set? It's easy enough for the linters to choose a specific one if they care or do an `#any?`, or similar, when they don't.

Reply:
During development I saw that the first index is always the character "*" and the second, in my case "pt", would always be correct.

https://cucumber.io/docs/gherkin/reference/#keywords

(en)
then => ['*', 'Então', 'Entao']

In this case the word "Então" would be spelled correctly.

I checked in a few more dialects and realized that this pattern apparently applies to the others as well.

Even so, for compatibility purposes we can only remove the "*" and compare it with the others, I will do it that way.